app/models/conversion_host.rb

```diff
@@ -252,6 +252,11 @@ def find_credentials(auth_type = 'v2v')
     if authentication.blank?
       res = resource.respond_to?(:authentication_type) ? resource : resource.ext_management_system
       authentication = res.authentication_type('ssh_keypair') || res.authentication_type('default')
+      if authentication
+        warning_message = "Unable to find v2v authentication for conversion host: #{name}. "\
+                          "Defaulting to authentication: #{authentication.name}/#{authentication.class}."
```

Comment:
Does this really warrant a warning?  Isn't this expected behavior for existing conversion hosts?

Reply:
Want me to change it to info? Or do you think this is more of a debug message?

Reply:
Is it even something that needs to be logged?  I thought this was an entirely valid situation to be in until we copy all resource authentications to the conversion host.

Reply:
I want it for debugging at least after some of the recent issues.

Reply:
@agrare I've changed it to a debug message.

Reply:
@djberg96 what issues are those?  I do not want to commit extra logging to master just to debug something.

Reply:
Issues where it was defaulting unexpectedly to the resource authentication because it wasn't finding the associated authentication.

We have tons of debug logging in core, not sure why it would be an issue.

Reply:
Not for debugging known issues though, usually adding the logging locally while you debug it is sufficient for that.  Is this something that you are unable to reproduce?